output/cloud/expv2/output.go

```diff
@@ -260,6 +341,18 @@ func (o *Output) handleFlushError(err error) {
 	})
 }
 
+func (o *Output) tracingEnabled() bool {
+	// TODO(lukasz): Check if k6 x Tempo is enabled
+	//
+	// We want to check whether a given organization is
+	// eligible for k6 x Tempo feature. If it isn't, we don't
+	// want to enable the traces output.
```

Comment:
```suggestion
	// eligible for k6 x Tempo feature. If it isn't, we may
	// consider to skip the traces output.
```
At this point, it could be not so relevant to skip them, so let's try it first then we will have a more accurate decision.